js/ui/text_box/ui.text_editor.mask.strategy.default.js

```diff
@@ -35,6 +35,10 @@ class DefaultMaskStrategy extends BaseMaskStrategy {
             this._handleBackspaceInput(event);
         }
 
+        if(event.originalEvent) {
```

Comment:
Why do we need this? When `originalEvent` can be not defined?

Reply:
Yes, standard char input has no original event. It filtered unnecessary _autoFillHandler requests

Reply:
Actually, this condition covers this case:
https://github.com/DevExpress/DevExtreme/blob/0efadc77b685650bf6876a7af282f0e08cf5e882/js/ui/text_box/ui.text_editor.mask.js#L380-L386

@DokaRus your thoughts?

Reply:
Looks like that shouldn't be a problem

Reply:
It shouldn't cause any problems.

Reply:
ok